src/sidebar/services/api.js

```diff
@@ -45,9 +45,9 @@ function stripInternalProperties(obj) {
 /**
  * Function which makes an API request.
  *
- * @template {Record<string, Param|Param[]>} Params
- * @template {object} Body
- * @template Result
+ * @template {Record<string, Param|Param[]>} [Params={}]
+ * @template [Body=void]
+ * @template [Result=void]
```

Comment:
This did me in for a bit until I revisited [Typescript JSDoc documentation for the `@template` tag](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#template). Syntax which indicates a default _value_ basically everywhere else here defines a default _type_. OK, I'll allow it :). But WRT `[Params={}]` I'm still a little wiggly. Does this mean "default type is an object" or "default type is an empty object" or...?